docs/dev/api-reference/legacy.rst

```diff
@@ -120,7 +120,7 @@ The API endpoint served at `upload.pypi.org/legacy/
 <https://upload.pypi.org/legacy/>`_ is Warehouse's emulation of the
 legacy PyPI upload API. This is the endpoint that tools such as `twine
 <https://twine.readthedocs.io/>`_ use to `upload distributions to PyPI
-<https://packaging.python.org/tutorials/distributing-packages/>`_.
+<https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#uploading-your-project-to-pypi>`_.
```

Comment:
I think we've tended to not use the locale in the url, leaving it up to the end user's browser to select. If unspecified, it's supposed to redirect.

Reply:
This is what I get for copying them from my browser 🙂 -- fixing.